<launch>
  <!--This was in core.launch-->
  <arg name="use_dynamics" default="1" />
  <arg name="use_external_renderer" default="0" />
  <arg name="ignore_collisions" default="false" />
  <arg name="render_stereo" default="0" />
  <arg name="run_challenges" default="1" />
  <arg name="gate_locations" default=""/>
  <arg name="render_quality" default="Fastest"/>

<!-- I dont think these lines are needed just to observe the AWS server remotely
<!--
  <group ns="/control_nodes">
    <!-- Start human teleoperation nodes -->
<!--    <node name="joy" pkg="joy" type="joy_node"/>
    <node name="keyboard" pkg="keyboard" type="keyboard"/>
    <node name="universal_teleop" pkg="universal_teleop" type="universal_teleop" output="screen">
      <rosparam file="$(find universal_teleop)/launch/example_input_map.yml"/>

      <remap from="output/rateThrust" to="/uav/input/rateThrust"/>
      <remap from="output/takeoff" to="/uav/input/takeoff"/>
      <remap from="output/land" to="/uav/input/land"/>
      <remap from="output/reset" to="/uav/input/reset"/>
    </node>
  </group>

  <!--Run RVIZ for visualization-->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find flightgoggles)/rviz/third_person_drone_follower.rviz" />

  <!-- Launch override settings. Must be placed at end of file to override params set in core launchfile. -->

  <!--For teleoperation, run dynamics in real time-->
  <param name="/use_sim_time" value="false" />
  <param name="/uav/flightgoggles_uav_dynamics/ignore_collisions" type="bool" value="$(arg ignore_collisions)" />

</launch>
